api/repositories/route_repository_test.go

```diff
@@ -403,7 +415,13 @@ var _ = Describe("RouteRepository", func() {
 				Expect(createdRouteRecord.Host).To(Equal(routeHost), "Route Host in record did not match input")
 				Expect(createdRouteRecord.Path).To(Equal(routePath), "Route Path in record did not match input")
 				Expect(createdRouteRecord.SpaceGUID).To(Equal(space.Name), "Route Space GUID in record did not match input")
-				Expect(createdRouteRecord.Domain).To(Equal(repositories.DomainRecord{GUID: domainGUID}), "Route Domain in record did not match created domain")
+
+				Expect(createdRouteRecord.Domain.GUID).To(Equal(domainGUID))
```

Comment:
just checking for the domain guid here is enough, the rest of the domain fields are already checked above. I would recommend getting rid of the rest of the expects on the domain here